resource-service/Dockerfile

```diff
@@ -43,7 +45,7 @@ RUN sed -i "s/version: develop/version: ${version}/g" docs/swagger.yaml
 RUN GOOS=linux go build -ldflags '-linkmode=external' -gcflags="${SKAFFOLD_GO_GCFLAGS}" -v main.go
 
 # Use a Docker multi-stage build to create a lean production image.
-FROM alpine:3.16 as production
+FROM alpine:3.15 as production
```

Comment:
is there a way to use 3.16 with the lib? We would introduce some open vulnerabilities with 3.15 :/

Reply:
by using the repo of 3.15 only for installing libgit2 it was possible to keep using 3.16 again